maskit/datasets/__init__.py

```diff
@@ -0,0 +1,36 @@
+from maskit.datasets.circles import circles
+from maskit.datasets.iris import iris
+from maskit.datasets.mnist import mnist
+from maskit.datasets.utils import Data
+
+
+def load_data(dataset: str, **kwargs):
+    """
+    Returns the data for the requested ``dataset``.
+
+    :param dataset (str): Name of the chosen dataset.
+        Available datasets are: iris, mnist and circles.
+    :param kwargs: Further arguments for dataset selection. Possible arguments:
+        train_size (int): Size of the training dataset
+        test_size (int): Size of the testing dataset
+        shuffle (boolean): if the dataset should be shuffled
+        classes (list[int]): which numbers of the mnist dataset should be included
+        wires (int): number of wires in the circuit
```

Comment:
With the default sphinx docstring style, there are no trailing ``(type)`` comments. For regular arguments, sphinx takes the type from the annotation. For ``kwargs`` it's fine, I've just adjusted on to match the actual type name.
```suggestion
    :param dataset: Name of the chosen dataset.
        Available datasets are: iris, mnist and circles.
    :param kwargs: Further arguments for dataset selection. Possible arguments:
        train_size (int): Size of the training dataset
        test_size (int): Size of the testing dataset
        shuffle (bool): if the dataset should be shuffled
        classes (list[int]): which numbers of the mnist dataset should be included
        wires (int): number of wires in the circuit
```